site/gatsby-site/server/types/incidents.ts

```diff
@@ -60,7 +60,14 @@ export const IncidentType = new GraphQLObjectType({
         flagged_dissimilar_incidents: { type: new GraphQLList(GraphQLInt) },
         nlp_similar_incidents: { type: new GraphQLList(NlpSimilarIncidentType) },
         reports: getListRelationshipConfig(ReportType, GraphQLInt, 'reports', 'report_number', 'reports', 'aiidprod'),
-        tsne: { type: TsneType }
+        tsne: { type: TsneType },
+        implicated_systems: {
```

Comment:
Instead of this, use `getListRelationshipConfig`, which is equivalent (unless I'm missing something) and more concise.